Tests/FuzzilliTests/LifterTest.swift

```diff
@@ -3095,4 +3095,35 @@ class LifterTests: XCTestCase {
         """
         XCTAssertEqual(actual, expected)
     }
-}
+
+    func testLoopNestedContinueLifting(){
```

Comment:
I'm wondering, does the label insertion work correctly if we also print line numbers? Did this test: https://github.com/googleprojectzero/fuzzilli/blob/f31876fff984ef7050adcdbe7f4c8bb3255ed8d0/Tests/FuzzilliTests/LifterTest.swift#L51 ever fail? Maybe you could add an extra test like this one that inserts labels but also prints line numbers?

Reply:
If this change breaks line numbering, maybe we could just add the label to the same line as the loop begin? So lift to something like `label1: for (...) {` instead?